packages/generator/src/gulpfileHelpers.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as path from "path";
+import _ from "lodash";
+
+import { diffRaml, NodeDiff } from "@commerce-apps/raml-toolkit";
+import { generatorLogger } from "./logger";
+
+export type RamlDiff = {
```

Comment:
Should this type be defined bt the raml-toolkit instead?

Reply:
I feel it is very specific to this use case. Why do you feel raml-toolkit is a better place for this?

Reply:
Because the eventual goal is to remove all of this logic from here :-p